docker-compose.yaml

```diff
@@ -17,6 +17,7 @@ services:
       - DB_VENDOR=postgres
       - DB_ADDR=keycloak-database
       - DB_USER=keycloak
+      - DB_USER=keycloak
```

Comment:
doubled up ?

Reply:
Huh? No idea where this came from.. Will fix..